k8s/zeppelin-server.yaml

```diff
@@ -29,7 +29,7 @@ data:
   # If you have your ingress controller configured to connect to `zeppelin-server` service and have a domain name for it (with wildcard subdomain point the same address), you can replace serviceDomain field with your own domain.
   SERVICE_DOMAIN: local.zeppelin-project.org:8080
   ZEPPELIN_K8S_SPARK_CONTAINER_IMAGE: spark:2.4.5
-  ZEPPELIN_K8S_CONTAINER_IMAGE: apache/zeppelin:0.9.0-SNAPSHOT
+  ZEPPELIN_K8S_CONTAINER_IMAGE: apache/zeppelin-interpreter:0.9.0-SNAPSHOT
   ZEPPELIN_HOME: /zeppelin
```

Comment:
based on Dockerfiles(`scripts/docker/zeppelin-interpreter/Dockerfile`、`scripts/docker/zeppelin-server/Dockerfile`), value of `ZEPPELIN_HOME` should be `/opt/zeppelin`